### PR TITLE
bug: change 'experience_level' to 'experienceLevel'

### PR DESCRIPTION
**Pull request by Nicolas-Maurer**
change 'experienceLevel' to 'experience_level' to match config.yaml otherwise desired experience wasn't filtered

### CODE REVIEW

File: src/aihawk_job_manager.py [hunk 1]
```diff
@@ -420,7 +420,7 @@ def get_base_search_url(self, parameters):
         url_parts = []
         if parameters['remote']:
             url_parts.append("f_CF=f_WRA")
-        experience_levels = [str(i + 1) for i, (level, v) in enumerate(parameters.get('experience_level', {}).items()) if
+        experience_levels = [str(i + 1) for i, (level, v) in enumerate(parameters.get('experienceLevel', {}).items()) if
```
Comment (surapuramakhil) [inline src/aihawk_job_manager.py:423]: @Nicolas-Maurer thanks for identifying it. 

upon reviewing the issue is with consistency in casing. Can you fix consistency? like making everything snake_case or camelCase

in both config yaml and also this file

**Review by surapuramakhil — COMMENTED** [review]
re-review test 😅

File: src/aihawk_job_manager.py (hunk 1 above)
Comment (feder-cr) [reply src/aihawk_job_manager.py:423]: All in snake case please

**Comment by feder-cr** [comment]
@Nicolas-Maurer please all in snake case

File: data_folder/config.yaml [hunk 2]
```diff
@@ -1,14 +1,14 @@
 remote: true
 
-experienceLevel:
+experience_level:
   internship: false
   entry: true
   associate: true
   mid-senior level: true
```
Comment (surapuramakhil) [inline data_folder/config.yaml:7]: its in kabab case - can you also these changes

**Review by surapuramakhil — CHANGES_REQUESTED** [review]
@Nicolas-Maurer there are few places where consistency is missing, can you kindly update those

sections of yaml:
date:
  all time: false
  month: false
  week: false
  24 hours: true
  
  and here mid-senior level: true

**Comment by himankpathak** [comment]
@feder-cr @Nicolas-Maurer  This PR broke `get_base_search_url` function. The date_mapping there still uses old config names.
File: `src/ai_hawk/job_manager.py`

**Pull request merged**